packages/adapter-vercel/index.d.ts

```diff
@@ -4,6 +4,7 @@ type Options = {
 	edge?: boolean;
 	external?: string[];
 	split?: boolean;
+	nodeVersion?: '16.x' | '14.x' | '12.x';
```

Comment:
why support Node 12? we don't anywhere else

how about Node 18?

Reply:
Node 18 isn't support by Vercel :disappointed: 

If Svelte doesn't support node 12.22, I can remove it